introduction.md

```diff
@@ -2,3 +2,6 @@ Hi there,
 I'm Daniel and eager to learn the possibilities of GIT.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?